prototype/src/main/webapp/chooseLocation.html

```diff
@@ -13,7 +13,7 @@ <h1>Meeting Locations: </h1>
     <button id="myButton">Send Event</button>
     <script type="text/javascript">
```

Comment:
(Nit) You could change this to:

```
<button id="myButton" onclick="location.href='/view-meeting.html'">Send Event</button>
```

So you don't need to write JavaScript in the HTML file. Since its only one line, it's probably fine, but I think this is related to '[Separation of Concerns](https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns)' in the style guide.

Reply:
Yup, I was thinking about that but I wasn't too sure. I'll change it (so it's similar to everything else as well).